DependencyInjection/Compiler/ResolveAdapterDefinitionPass.php

```diff
@@ -31,7 +31,16 @@ public function process(ContainerBuilder $container)
         $definition->setAbstract(false);
         $definition->setClass($adapterClasses[$container->getParameter('prometheus_metrics_bundle.type')]);
         if ('redis' === $container->getParameter('prometheus_metrics_bundle.type')) {
-            $definition->setArguments([$container->getParameter('prometheus_metrics_bundle.redis')]);
+            $redisArguments = $container->getParameter('prometheus_metrics_bundle.redis');
+
+            $prefix = $redisArguments['prefix'] ?? null;
+            if ($prefix) {
```

Comment:
I think, we should have here a strict comparison, or maybe `!empty($prefix)` call, to be a little be more type-safe.
P.S. Would you please also remove all those new lines as here I don't think they give any semantical sense.